container/Dockerfile

```diff
@@ -0,0 +1,111 @@
+# BUILDER
+FROM alpine:latest AS builder
+
+#build ARGs
+ARG _RADARE2_GIT_REPO="https://github.com/radareorg/radare2.git"
+ENV RADARE2_GIT_REPO=$_RADARE2_GIT_REPO
+
+ARG _KS_NAV_GIT_REPO="https://github.com/elisa-tech/ks-nav"
+ENV KS_NAV_GIT_REPO=$_KS_NAV_GIT_REPO
+ARG _KS_NAV_GIT_REPO_BRANCH=main
+ENV KS_NAV_GIT_REPO_BRANCH=$_KS_NAV_GIT_REPO_BRANCH
+
+# install build dependecies
+RUN apk add --no-cache autoconf automake build-base git
+RUN apk add --no-cache go
+RUN apk add --no-cache linux-headers
+
+#build app
+WORKDIR /build
+
+# Build radare2
+RUN git clone ${RADARE2_GIT_REPO} /build/radare2 && cd /build/radare2 && git checkout ecd8d00aad289abf8f850b45d5f6c2e6b0c15c0f
```

Comment:
It would be good to have an `ARG` for the revision. Maybe also some short explanation why we don't use an official release.

Reply:
Got it, valuable suggestion. Thanks.